content/2.components/lens.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Lens
+description: A lens component to zoom into images, videos, or practically anything.
+navBadges:
+  - value: New
+    type: lime
+---
+
+::ComponentLoader{label="Preview" componentName="LensDemo" type="examples" id="lens"}
+::
+
+## Examples
+
+### Basic
+
+::ComponentLoader{label="Preview" componentName="LensDemo" type="examples" id="lens"}
+::
+
+### Static
+
+Lens are static in center
+
+::ComponentLoader{label="Preview" componentName="StaticLensDemo" type="examples" id="lens"}
+::
+
+## API
+
+| Prop Name | Type     | Default | Description                                                     |
+| --------- | -------- | ------- | --------------------------------------------------------------- |
+| `zoomFactor`   | `number` |   `1.5`   | The magnification factor for the lens.               |
+| `lensSize` | `number` |  `170` | The diameter of the lens in pixels. |
+| `position` | `{ x: number, y: number }` | `{ x: 200, y: 150 }` | The static position of the lens (when isStatic is true). |
+| `isStatic` | `boolean` | `false` | If true, the lens stays in a fixed position; if false, it follows the mouse. |
+| `hovering` | `boolean` | `"false"` | External control for the hover state. |
+
+## Component Code
+
+You can copy and paste the following code to create these components:
+
+::CodeViewer{filename="Lens.vue" language="vue" componentName="Lens" type="ui" id="lens"}  
```

Comment:
Please include `Rays` components also here, You can use ::code-group and use ::CodeViewerTab to add multiple codes.

Reply:
changes are commited